src/views/portals/outings-stats/outings-stats.js

```diff
@@ -2,12 +2,23 @@ import d3 from '@/js/libs/d3';
 
 export class Histogram {
   constructor(data, htmlElement, getX) {
-    this.data = data;
-    this.htmlElement = htmlElement;
+    this._data = data;
+    this._htmlElement = htmlElement;
     this._getX = getX;
     this._xTickLabel = (value) => value;
     this._yTickLabel = (value) => value;
     this._getY = () => 1;
+
+    // set the dimensions and margins of the graph
+    this._margin = { top: 10, right: 30, bottom: 50, left: 40 };
```

Comment:
could be a const

Reply:
I also could have done a static property, with an immutable object. But as I don't know how it'll be used later (I bet one day or another we'll make this values variables), I assume the best choice for now is it keep it simple.